src/protofiles/Value.proto

```diff
@@ -43,4 +45,8 @@ message Value {
         bool m_timeStampValid = 18;
         int32 m_transcieverClass = 19;
     }
+    
+    message Waypoints {
+        repeated double waypoints = 1;
```

Comment:
Waypoint should be a list of pairs of doubles representing lat/lon.
Can you make a message type defined inside "message Waypoints"  called GpsCoord and give it double latitude and double longitude, and then make Wapoints have a list of these instead

Reply:
Actually, you can create the GpsCoord message as a subtype of Waypoints, or just create it outside if you think its simpler to read/use.